pastel/pastel.go

```diff
@@ -33,4 +33,10 @@ type Client interface {
 	// Verify verifies signed data by the given its signature and pastelID, if successful returns true.
 	// Command `pastelid verify "text" "signature" "PastelID"`.
 	Verify(ctx context.Context, data []byte, signature, pastelID string) (ok bool, err error)
+	// ActTickets returns activated art tickets.
+	// Command `tickets list act`.
+	ActTickets(ctx context.Context, actType ActTicketType, minHeight int) (ActTickets, error)
+	// ActTickets returns art registration tickets.
+	// Command `tickets get <txid>`.
+	RegTicket(ctx context.Context, regTxid string) (RegTicket, error)
```

Comment:
nit: can add action prefix in these API? this can help prevent confusing when code volume get bigger.
like:
- QueryActTickets()
- GetRegTicket()

Reply:
1- @levkoburburas  suggested on naming ActivationTickets as ActTicket and keeping it consistent throughout the project. 
2- Also suggested to avoid words like `GET`  as prefixes to function names

we only have one Activation ticket & Registration ticket... AFAIK